charts/apps/values.yaml

```diff
@@ -1,12 +1,19 @@
 project: ""
 
-source:
+monitoring:
   repoURL: https://github.com/DiamondLightSource/graph-federation
   targetRevision: HEAD
-  monitoring:
-    path: charts/monitoring
-  graph:
-    path: charts/graph
+  path: charts/monitoring
+
+graph:
+  repoURL: https://github.com/DiamondLightSource/graph-federation
+  targetRevision: HEAD
+  path: charts/graph
+
+schema:
+  repoURL: ghcr.io/diamondlightsource/graph-federation-supergraph
+  targetRevision: 0.1.1
```

Comment:
In my mind we were going to use `x` and manually click "sync" in ArgoCD, but this is far more GitOps-y and keeps state in one place - I like it.

Perhaps we should turn on auto-sync in the stable chart and have the `graph` source point to a fixed release number too. What do you think?

Reply:
Yeah, that's what I thought we were gna do, that's the reason I had targetVersion and stableVersion. we can make the graph point to the stableVersion and turn on auto-sync?

Reply:
Wasn't what was in my head, but is probably the better route, I'd say go that way